src/ray/gcs/store_client/test/observable_store_client_test.cc

```diff
@@ -26,8 +26,8 @@ namespace gcs {
 class ObservableStoreClientTest : public StoreClientTestBase {
  public:
   void InitStoreClient() override {
-    store_client_ = std::make_shared<ObservableStoreClient>(
-        std::make_unique<InMemoryStoreClient>(*(io_service_pool_->Get())));
+    store_client_ =
+        std::make_shared<ObservableStoreClient>(std::make_unique<InMemoryStoreClient>());
```

Comment:
why this change?

Reply:
https://github.com/ray-project/ray/blob/7579716cd1cf9a3f6ed19c26f90c71924049dcb0/src/ray/gcs/store_client/in_memory_store_client.h#L35

Thanks for the fix! Now I'm confused how it compiles in the past.

Reply:
> Now I'm confused how it compiles in the past.

I think that’s because observable_store_client_test.cc is not actually included in the current build targets.
https://github.com/ray-project/ray/blob/7579716cd1cf9a3f6ed19c26f90c71924049dcb0/BUILD.bazel#L1516-L1519